dali/operators/math/normalize/normalize.cc

```diff
@@ -120,7 +120,11 @@ void Normalize<CPUBackend>::FoldMeans() {
   assert(mean_.ntensor() > 0);
   SumSamples(view<float>(mean_));
   // calculate the normalization factor in double, then cast to float
-  float den = static_cast<float>(1.0 / ReducedVolume(data_shape_, make_span(axes_)));
+  auto v = ReducedVolume(data_shape_, make_span(axes_));
+  if (v == 0) {
```

Comment:
This would never happen, unless the data was empty.

Reply:
I can imagine that it could be and it this case it still should work without crash.